src/wechaty.spec.ts

```diff
@@ -149,6 +149,38 @@ test('on(event, Function)', async t => {
 
 })
 
+test('use plugin', async (t) => {
+  let result = ''
+
+  const myGlobalPlugin = function () {
+    return function (this: Wechaty) {
+      this.on('message', () => (result += 'FROM_GLOBAL_PLUGIN:'))
+    }
+  }
+
+  const myPlugin = function () {
+    return function (this: Wechaty) {
+      this.on('message', () => (result += 'FROM_MY_PLUGIN:'))
+    }
+  }
+  Wechaty.use(myGlobalPlugin())
```

Comment:
```ts
class WechatyTest extends Wechaty {}
WechatyTest.use(myGlobalPlugin())
```

This would be better because it will not modify the global `Wechaty` class.

Reply:
agree